catalog-info.yaml

```diff
@@ -366,3 +366,44 @@ spec:
           access_level: BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: beats-agent-serverless-tests
+  description: "Beats serverless tests from elastic-agent repository"
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-agent-serverless-tests
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: platform-ingest
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: beats-agent-serverless-tests
+      description: "Beats serverless tests from elastic-agent repository"
+    spec:
+      pipeline_file: ".buildkite/serverless.beats.tests.yml"
+      maximum_timeout_in_minutes: 180
+      provider_settings:
+        trigger_mode: none # don't trigger jobs from github activity
+      repository: elastic/elastic-agent
+      teams:
+        ingest-fp:
+          access_level: MANAGE_BUILD_AND_READ
+        observablt-robots:
+          access_level: MANAGE_BUILD_AND_READ  
+        everyone:
+          access_level: BUILD_AND_READ
```

Comment:
Do we mean to enable this access?

Reply:
I took the lead and do https://github.com/elastic/elastic-agent/pull/7350 (in addition to some other changes I needed)